### DISCUSSION TITLE
[Feature] Add shortlink support (or integration with yourls / similar url shortener) for shared links

### DISCUSSION THREAD

**Discussion by kkoshelev** [immich-app/immich#4730] [Feature Request]
### The feature

Currently it is difficult to share links with people because they are simply too long. It should be an option to create something shorter. I currently have to do extra step to copy&paste link to my own yourls before sharing with someone over chat or text message.

https://pics.example.com/share/_euCbCTjkh23k4j2h4k23iCu73BB5SAcq7uvQ23-QfFY1J1cnU9OsXUOshrVoi9Mr3wWU
vs https://pics.example.com/go/my-fancy-album.

Or use existing shortener, see below.
https://yourls.org/ is self-hosted url shortener. It would be nice to have an integration done.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by kkoshelev** [comment]
@jrasm91 due to resent works related to shared links

**Comment by mxz94** [comment]
I agree

**Comment by scottrbaxter** [comment]
+1 for other self-hosted services like yourls.org integration!

**Comment by zkvvoob** [comment]
Is this still being considered?

**Reply by bo0tzz** [reply]
Yes, otherwise the feature request would be closed.

**Reply by jrasm91** [reply]
#19999

**Comment by dahamsta** [comment]
YOURLS user here, the Immich links are indeed very cumbersome, I'd love to see shorter links natively or integration with YOURLS.

**Comment by jrasm91** [comment]
Implemented in #19999
